examples/duel/src/main/webapp/js/todos/controller.js

```diff
@@ -85,20 +57,28 @@ var todos = todos || {};
 		} else {
 			todos.model.remove(id);
 		}
-		refreshAll();
+		refreshView();
+	}
+
+	function reset(input, id) {
+		var task = todos.model.find(id);
+		if (task) {
+			input.value = task.title;
+		}
 	}
 
 	/*-- export public interface -------------------------------*/
 
 	// event handlers
 	todos.actions = {
-		addBlur: function () {
-			add(this);
-		},
-
 		add_keypress: function (e) {
-			if (e.keyCode === ENTER_KEY) {
-				add(this);
+			switch (e.keyCode) {
```

Comment:
Why change this from an `if` statement to a `switch`? Seems still easier to simply do it as before, no?